### PR TITLE
Add identities updating  option in dashboard

### PR DESCRIPTION
**Pull request by ranatrk**
Identities
- Add/register new identity from dashboard settings
- View identities in dashboard settings
- Delete identity from dashboard
- Change default identity used from dashboard (will be set in ` j.core.identity.me`)

Wallets
- Add update known trustline option for wallets from dashboard

Screenshot from 2020-07-20 16-04-31
Screenshot from 2020-07-20 16-04-39
Screenshot from 2020-07-20 16-05-24

Screenshot from 2020-07-20 16-05-05

### CODE REVIEW

File: jumpscale/packages/admin/actors/admin.py [hunk 1]
```diff
@@ -77,5 +79,74 @@ def set_explorer(self, explorer_type: str) -> str:
...
+            return j.data.serializers.json.dumps({"data": f"{identity_instance_name} doesn't exist"})
+
+    @actor_method
+    def set_identity(self, identity_instance_name: str) -> str:
+        identity_names = j.core.identity.list_all()
+        if identity_instance_name in identity_names:
+            j.core.identity.set_default(identity_instance_name)
+
+            # update our solutions
+            j.sals.reservation_chatflow.get_solutions_explorer()
```
Comment (dmahmouali) [inline jumpscale/packages/admin/actors/admin.py:120]: now `get_solutions_explorer()` just return the reservation data, to update local call `update_local_reservations()`

Comment (ranatrk) [reply jumpscale/packages/admin/actors/admin.py]: updated here https://github.com/threefoldtech/js-sdk/pull/261/commits/9b893d50386d152363927bdd0f437f0062086ec3

File: jumpscale/packages/admin/frontend/components/settings/AddIdentity.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        },
+        explorer_labels:["Main Network","Test Network"]
+      }
+    },
+    methods: {
+        submit () {
+            this.loading = true
+            this.error = null
+            if(!this.form.instance_name || !this.form.tname || !this.form.email || !this.form.words || !this.form.explorer_type){
+                this.alert("All fields required", "error")
```
Comment (ahsayde) [inline jumpscale/packages/admin/frontend/components/settings/AddIdentity.vue:37]: please change this to 
```this.error = "All fields required" ```

File: jumpscale/packages/admin/frontend/components/settings/AddIdentity.vue [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                this.loading = false
+            }
+            else{
+                this.$api.identities.add(this.form.instance_name, this.form.tname, this.form.email, this.form.words, this.explorers[this.form.explorer_type].type).then((response) => {
+                    responseMessage = JSON.parse(response.data).data
+                    if(responseMessage == "Identity with the same instance name already exists"){
+                        this.alert(responseMessage, "error")
```
Comment (ahsayde) [inline jumpscale/packages/admin/frontend/components/settings/AddIdentity.vue:44]: please change this to
this.error = "All fields required"

File: jumpscale/packages/admin/frontend/components/settings/AddIdentity.vue (hunk 2 above)
Comment (ranatrk) [reply jumpscale/packages/admin/frontend/components/settings/AddIdentity.vue]: updated in https://github.com/threefoldtech/js-sdk/pull/261/commits/66f2c240868f4fc596152d45ba4e7659d172d343

File: jumpscale/packages/admin/frontend/components/settings/AddIdentity.vue (hunk 3 above)
Comment (ranatrk) [reply jumpscale/packages/admin/frontend/components/settings/AddIdentity.vue]: updated in https://github.com/threefoldtech/js-sdk/pull/261/commits/66f2c240868f4fc596152d45ba4e7659d172d343

**Pull request merged**